extension/src/XliffEditor/XliffEditorPanel.ts

```diff
@@ -187,7 +187,13 @@ export class XliffEditorPanel {
             xlfDocument.original
         );
         filteredXlf._path = xlfDocument._path;
-        filteredXlf.transunit = xlfDocument.transunit.filter(u => (u.targets[0].translationToken !== undefined) || (u.hasCustomNote(CustomNoteType.RefreshXlfHint) || filter === "all"));
+        if (filter === "differently-translated") {
+            filteredXlf.transunit = xlfDocument.differentlyTranslatedTransunits();
+            filteredXlf.transunit.sort((a, b) => (a.source > b.source) ? 1 : ((b.source > a.source) ? -1 : 0));
```

Comment:
I didn't know that the sort function was "in-place"! 😆

Reply:
it's neat and scary at the same time :D

Reply:
I guess I really should copy the array to be sure I'm not modifying the orginial.,,

Reply:
> I guess I really should copy the array to be sure I'm not modifying the orginial.,,

But this is an copy of the array, isn't it? just that the trans-units are the same...

Reply:
yep, it's just me not trusting JS or rather not knowing for sure. I need to test it out at some point and maybe I'll learn :)